master/pkg/check/validate.go

```diff
@@ -60,6 +60,10 @@ func validate(v reflect.Value, path interface{}) []error {
 		}
 	case reflect.Struct:
 		for i := 0; i < v.NumField(); i++ {
+			if !v.Field(i).CanInterface() {
```

Comment:
So when checking structs, we were previously checking exported and unexported fields, however when you try to check an unexported field you hit an error. Previously this wasn't an issue because none our structs in the experiment config had unexported fields. This check skips the unexported fields in `v1.Pod`.

Reply:
blocking: I see. IIUC, it tries to access unexported fields, which is not possible for the external package `v1.Pod`. Could you add one comment like "// Skip unexported fields."?

Reply:
done